lib/checks/aria/allowed-attr.js

```diff
@@ -7,12 +9,22 @@ var attr, attrName, allowed,
 if (!role) {
 	role = axe.commons.aria.implicitRole(node);
 }
+
 allowed = axe.commons.aria.allowedAttr(role);
+
+if (Object.keys(options).length) {
```

Comment:
You can simplify this.
```js
if (Array.isArray(options[role])) {
  allowed = axe.utils.uniqueArray(options[role].concat(allowed));
}
```

Reply:
Done. I also removed a duplicate `to-array` file in axe.commons. It was almost exactly the same code duplicated, we don't want to maintain both if we don't have to.

Reply:
Oops, I was confused by this comment...I've made the change and added it to this PR. Thanks!